### PR TITLE
V7.2.0

### PR DESCRIPTION
**Pull request by rmarinleal**
* Replacing .env file with DB configs
* Creating a configController
* Simulator v3

### CODE REVIEW

File: test/PmtWebDriver.php [hunk 1]
```diff
@@ -16,11 +16,11 @@ class PmtWebDriver extends RemoteWebDriver
      * Override method to increase the default timeouts
      *
      * @param int $timeout_in_second
-     * @param int $interval_in_millisecond
+     * @pgit push orioginaram int $interval_in_millisecond
```
Comment (pgarcess) [inline test/PmtWebDriver.php]: git push ¿?

File: controllers/front/config.php [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+     * Read PTM configs
+     *
+     * @throws PrestaShopDatabaseException
+     */
+    public function getMethod()
+    {
+        $sql_content = 'select * from ' . _DB_PREFIX_. 'pmt_config';
+        $dbConfigs = Db::getInstance()->executeS($sql_content);
+
+        // Convert a multimple dimension array for SQL insert statements into a simple key/value
```
Comment (romeritoCL) [inline controllers/front/config.php]: Remove the comment

File: paylater.php [hunk 3]
```diff
@@ -31,10 +31,36 @@ class Paylater extends PaymentModule
...
     /**
-     * @var installErrors
+     * @var array
      */
     public $installErrors = array();
 
+    /**
+     * @var array
+     */
+    public $dotEnvError = null;
```
Comment (pgarcess) [inline paylater.php]: Ya no la usamos

File: test/PmtWebDriver.php (hunk 1 above)
Comment (romeritoCL) [inline test/PmtWebDriver.php]: Pum

File: paylater.php (hunk 3 above)
Comment (rmarinleal) [reply paylater.php]: resolved

File: controllers/front/config.php (hunk 2 above)
Comment (rmarinleal) [reply controllers/front/config.php]: deleted

File: test/PmtWebDriver.php (hunk 1 above)
Comment (rmarinleal) [reply test/PmtWebDriver.php]: resolved

**Pull request merged**